assets/dex/stormtrade.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "address": "UQB2OpR9gqb6lmiinABLjt1ZDGjHV6XO9S-DqzEv2OKIMkUo",
```

Comment:
Let's keep all addresses in all JSONs in the same format.

So this should be replaced with 

`EQB2OpR9gqb6lmiinABLjt1ZDGjHV6XO9S-DqzEv2OKIMhjt`